internal/app/feed/feed_action_assignees.go

```diff
@@ -80,7 +80,7 @@ func (f *ActionAssigneeFeed) writeRows(ctx context.Context, exporter Exporter, r
 			j = len(rows)
 		}
 		var actionIDs []string
-		for k := i; k < (i + j); k++ {
```

Comment:
This is just getting all the IDs for the current batch right?

Can we just do `for action, _ := range rows[i:j]`

Reply:
Also can you please add a test case 🙏

Reply:
Added a test and updated.